src/mmw/js/src/analyze/views.js

```diff
@@ -746,10 +746,9 @@ var PointSourceResultView = AnalyzeResultView.extend({
 
 var CatchmentWaterQualityResultView = AnalyzeResultView.extend({
     onShow: function() {
-        var desc = 'Delaware River Basin only: ' +
-        '<a target="_blank" ' +
-        'href="https://www.arcgis.com/home/item.html?id=260d7e17039d48a6beee0f0b640eb754"​>' +
-        'Stream Reach Assessment Tool</a> model estimates',
+        var desc = 'Delaware River Basin only: <a target="_blank" ' +
+            'href="https://www.arcgis.com/home/item.html?id=260d7e17039d48a6beee0f0b640eb754"​>' +
```

Comment:
Since we're working on indentation, consider lining this up with the start of the string above, like this:

```js
        var desc = 'Delaware River Basin only: <a target="_blank" ' +
                   'href="https://www.arcgis.com/home/item.html?id=260d7e17039d48a6beee0f0b640eb754"​>' +
                   'Stream Reach Assessment Tool</a> model estimates',
            chart = null;
```